packages/debian/download_packages.py

```diff
@@ -111,12 +111,12 @@ def add_external_repositories(salt_version: str) -> None:
             'name': 'saltstack',
             'key': urllib.parse.urljoin(
                 'https://repo.saltstack.com/',
-                'apt/ubuntu/18.04/amd64/{}/SALTSTACK-GPG-KEY.pub'.format(
+                'apt/ubuntu/18.04/amd64/archive/{}/SALTSTACK-GPG-KEY.pub'.format(
                     salt_version
                 )
             ),
             'source': 'deb http://repo.saltstack.com/apt/ubuntu/'\
-                      '18.04/amd64/{} bionic main'.format(salt_version)
+                      '18.04/amd64/archive/{} bionic main'.format(salt_version)
```

Comment:
Ditto .. i think

Reply:
Indeed, I used the full version locally and wanted to check with @slaperche-scality to see if I could commit it. Just pushed a fix!